LEGO1/lego/legoomni/src/race/raceskel.cpp

```diff
@@ -1,13 +1,24 @@
 #include "raceskel.h"
 
+#include "carrace.h"
+#include "legoworld.h"
+#include "misc.h"
+
 #include <assert.h>
 
 DECOMP_SIZE_ASSERT(RaceSkel, 0x178)
 
-// STUB: LEGO1 0x100719b0
+// FUNCTION: LEGO1 0x100719b0
+// FUNCTION: BETA10 0x100f1240
 RaceSkel::RaceSkel()
 {
-	// TODO
+	m_animPosition = 0.0f;
+}
+
+// FUNCTION: LEGO1 0x10071ad0
+RaceSkel::~RaceSkel()
+{
+	// Removing this empty destructor leads to a mismatch in RaceSkel::`scalar deleting destructor'
```

Comment:
I am a little unsure about this part, but I couldn't get anything close to a match with a purely synthetic destructor. Declaring an empty destructor in `raceskel.h` was affected by the same problem.

Reply:
Yes this is correct. You can remove the comment. Sometimes they just defined an empty explicit constructor in this way